packages/functional-tests/lib/accountManager.ts

```diff
@@ -0,0 +1,145 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import crypto from 'crypto';
+import { Credentials } from './targets';
+import { BaseTarget } from './targets/base';
+
+const BLOCKED_EMAIL_PREFIX = 'blocked{id}';
+const BOUNCED_EMAIL_PREFIX = 'bounced{id}';
+const FORCED_PWD_EMAIL_PREFIX = 'forcepwdchange{id}';
+const SIGNIN_EMAIL_PREFIX = 'signin{id}';
+const SIGNUP_REACT_EMAIL_PREFIX = 'signup_react{id}';
+const SYNC_EMAIL_PREFIX = 'sync{id}';
+
+type EmailPrefix =
+  | typeof BLOCKED_EMAIL_PREFIX
+  | typeof BOUNCED_EMAIL_PREFIX
+  | typeof FORCED_PWD_EMAIL_PREFIX
+  | typeof SIGNIN_EMAIL_PREFIX
+  | typeof SIGNUP_REACT_EMAIL_PREFIX
+  | typeof SYNC_EMAIL_PREFIX;
+
+type Account = {
+  email: string;
+  password: string;
+};
+
+export class AccountManager {
```

Comment:
Alternative Name Possibilities:
TestAccountCoordinator

Reply:
Opting for TestAccountTracker (see https://github.com/mozilla/fxa/pull/16739/commits/e5811b43eca6400a6a1bda82a49b449645465b50)